server/game/GameActions/RemoveTokenAction.js

```diff
@@ -17,20 +18,20 @@ class RemoveTokenAction extends CardGameAction {
         if(this.amount === 1) {
             token = '+1 power counter';
         }
-        this.effectMsg = 'remove ' + this.amount + ' ' + (this.type === 'power' ? token : this.type) + ' on {0}';
+        this.effectMsg = 'remove ' + (this.all ? 'all' : this.amount) + ' ' + (this.type === 'power' ? token : this.type) + ' from {0}';
+    }
+
+    computeAmount(card) {
```

Comment:
getAmount is more consistent naming with other things of this nature

Reply:
Fixed.